### ISSUE TITLE
Use auxiliary views for the buttons

### ISSUE THREAD

**Issue by Kentzo** [Kentzo/ShortcutRecorder#116]
Currently the control draws its auxiliary elements in its implementation of drawRect. It also handles dirtying and accessibility. Accessibility in particularly does not work very well in such approach (e.g. a user cannot navigate to these aux elements).

Both clear and delete buttons should be implemented as either custom views or NSImageView's (if it's possible to set up accessibility properly).

If custom subview path is taken, it should implement _both_ -updateLayer and -drawRect:
